digiwf-integrations/digiwf-s3-integration/digiwf-s3-integration-core/src/main/java/io/muenchendigital/digiwf/s3/integration/application/port/in/CleanUpExpiredFilesInPort.java

```diff
@@ -0,0 +1,8 @@
+package io.muenchendigital.digiwf.s3.integration.application.port.in;
+
+import org.springframework.transaction.annotation.Transactional;
+
+public interface CleanUpExpiredFilesInPort {
+  @Transactional
```

Comment:
Why Transactional here and not on other In ports like `CleanUpUnusedFoldersInPort`